src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlDynamicKey.cs

```diff
@@ -1,24 +1,22 @@
-﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
-using JuliusSweetland.OptiKey.Enums;
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - AllRights Reserved
 using System.Collections.Generic;
 using System.Xml.Serialization;
 
 namespace JuliusSweetland.OptiKey.Models
 {
-    public class XmlDynamicKey : XmlKey
```

Comment:
In order to free up the name "DynamicKey" for my new code, I renamed the old version to "MultiStepKey". It would be ideal to deprecate "MultiStepKey" when the new more advanced "DynamicKey" is released.